updatehub/src/object/installer/copy.rs

```diff
@@ -42,6 +42,32 @@ impl Installer for objects::Copy {
         let target_path = self.target_path.strip_prefix("/").unwrap_or(&self.target_path);
         let source = download_dir.join(sha256sum);
 
+        match self.install_if_different {
+            Some(ref rule) => {
+                match utils::fs::mount_map(&device, filesystem, mount_options, |path| {
+                    let dest = path.join(&target_path);
+                    fs::File::open(&dest).map_err(Error::from).and_then(|mut handle| {
+                        super::check_if_different(&mut handle, rule, &self.sha256sum)
+                    })
```

Comment:
```suggestion
                    fs::File::open(&dest.join(&target_path))
                        .map_err(Error::from).and_then(|mut h| {
                        super::check_if_different(&mut h, rule, &self.sha256sum)
                    })
```
Does this looks nicer? it is closer to other similar code.